contents/docs/experiments/running-experiments-without-feature-flags.mdx

```diff
@@ -0,0 +1,238 @@
+
+This step is not necessary if you are using your own feature flag library.
+
+## Step 3: Add the feature flag property to your events 
+
+In order to attribute events to their correct experiment variants, you'll need include a `$feature/experiment_feature_flag_key: variant-name` property when submitting your events. You only need to do this for events that you'd like to track in your experiment.
+
+This is similar to the [same step](/docs/experiments/manual#step-2-server-side-only-add-the-feature-flag-property-to-your-events) when setting up regular experiments. However, the key difference is that now you'll need to include this property when submitting events from all SDKs, and not just server-side SDKs.
+
+<MultiLanguage>
+
+```javascript
+posthog.capture('event_name_of_your_goal_metric', {
+    "$feature/experiment-feature-flag-key": "variant-name"
+})
+```
+
+```react
+posthog.capture('event_name_of_your_goal_metric', {
+    "$feature/experiment-feature-flag-key": "variant-name"
+})
+```
+
+```react-Native
+posthog.capture('event_name_of_your_goal_metric', {
+    "$feature/experiment-feature-flag-key": "variant-name"
+})
+```
+
+```Android
+PostHog.with(this)
+       .capture("event_name_of_your_goal_metric", new Properties()
+                                        .putValue("$feature/experiment-feature-flag-key", "variant-name"));
+```
+
+```iOS
+posthog.capture("event_name_of_your_goal_metric", properties: ["$feature/experiment-feature-flag-key": "variant-name"])
+```
+
+```node
+client.capture({
+    distinctId: 'distinct_id',
+    event: 'event_name_of_your_goal_metric',
+    properties: {
+        '$feature/experiment-feature-flag-key': 'variant-name'
+    },
+})
+```
+
+```python
+posthog.capture(
+    'distinct_id',
+    'event_name_of_your_goal_metric',
+    {
+        '$feature/experiment-feature-flag-key': 'variant-name'
+    }
+)
+```
+
+```php
+PostHog::capture(array(
+  'distinctId' => 'distinct_id',
+  'event' => 'event_name_of_your_goal_metric',
+  'properties' => array(
+    '$feature/experiment-feature-flag-key'  = 'variant-name'
+  )
+));
+```
+
+```ruby
+posthog.capture({
+    distinct_id: 'distinct_id',
+    event: 'event_name_of_your_goal_metric',
+    properties: {
+        '$feature/experiment-feature-flag-key': 'variant-name',
+    }
+})
+```
+
+```go
+client.Enqueue(posthog.Capture{
+  DistinctId: "distinct_id",
+  Event:      "event_name_of_your_goal_metric",
+  Properties: posthog.NewProperties().
+    Set("$feature/experiment-feature-flag-key", "variant-name"),
+})
+```
+
+```java
+posthog.capture(
+  "distinct_id",
+  "event_name_of_your_goal_metric",
+  new HashMap<String, Object>() {
+    {
+      put("$feature/experiment-feature-flag-key", "variant-name");
+    }
+  }
+);
+```
+
+</MultiLanguage>
+
+## Step 4: Send the `$feature_flag_called` event
```

Comment:
Maybe add a line that you need to send this only for trend experiments?

Since on funnel experiments exposure is determined by first step in the funnel

Reply:
Nice. Didnt know this